deepchem/feat/material_featurizers/lcnn_featurizer.py

```diff
@@ -0,0 +1,656 @@
+
+  Examples
+  ----------
+  The input format for primitive cell Template is stored in a file named
+
+  [comment]
+  [ax][ay][az][pbc]
+  [bx][by][bz][pbc]
+  [cx][cy][cz][pbc]
+  [number of spectator site type][number of active site type]
+  [os1][os2][os3]
+  [number sites]
+  [site1a][site1b][site1c][site type]
+  [site2a][site2b][site2c][site type]
+
+  - ax,ay, ... are cell basis vector
+  - pbc is either T or F indication of the periodic boundary condition
+  - os# is the name of the possible occupation state (interpretted as string)
+  - site1a,site1b,site1c are the scaled coordinates of site 1
+  - site type can be either S1, S2, ... or A1, A2,... indicating spectator
+  ...
+  Example:
+  #Primitive Cell
+  2.81859800e+00  0.00000000e+00  0.00000000e+00 T
+  -1.40929900e+00  2.44097800e+00  0.00000000e+00 T
+  0.00000000e+00  0.00000000e+00  2.55082550e+01 T
+  1 1
+  -1 0 1
+  6
+  0.00000000e+00  0.00000000e+00  9.02210000e-02 S1
+  6.66666666e-01  3.33333333e-01  1.80442000e-01 S1
+  3.33333333e-01  6.66666666e-01  2.69674534e-01 S1
+  0.00000000e+00  0.00000000e+00  3.58978557e-01 S1
+  6.66666666e-01  3.33333333e-01  4.49958662e-01 S1
+  3.33333333e-01  6.66666666e-01  5.01129144e-01 A1
+
+
+  The input format for a data point Structure (Configuration of Atoms):
+  ...
+  [ax][ay][az]
+  [bx][by][bz]
+  [cx][cy][cz]
+  [number sites]
+  [site1a][site1b][site1c][site type][occupation state if active site]
+  [site2a][site2b][site2c][site type][occupation state if active site]
+  ...
+  - property value indicates the trained value. It must start with #y=...
+  ...
+  Example:
+  2.81859800e+00  0.00000000e+00  0.00000000e+00
+  -1.40929900e+00  2.44097800e+00  0.00000000e+00
+  0.00000000e+00  0.00000000e+00  2.55082550e+01
+  6
+  0.000000000000  0.000000000000  0.090220999986 S1
+  0.500000499894  0.622008360788  0.180442000011 S1
+  0.999999500106  0.666666711253  0.270892474701 S1
+  0.000000000000  0.000000000000  0.361755713893 S1
+  0.500000499894  0.622008360788  0.454395429618 S1
+  0.000000000000  0.666667212896  0.502346789304 A1 1
+  ...
+  Python Script:
+  ...
+  >>> template_file_path = os.join.path(data_dir , "input.in")
+  >>> Featurizer =  np.around(6.00,2)
+  >>> Data_point_path = "Structure1.txt"
+  >>> Data_point_text = open(Data_point_path).read()
+  >>> graph_ob = Featurizer._featurize(Data_point_text)
+  >>> print(graph_ob)
+  """
+
+  def __init__(self, cutoff, template):
+    """
+    Parameters
+    ----------
+    cutoff: cutoff of radius for getting local environment.Only
+        used down to 2 digits.
+
+    template: Template primitive stucture file path
+    """
+    self.cutoff = np.around(cutoff, 2)
+    self.setup_env = SiteEnvironments.Load(template, cutoff)
+
+  def _featurize(self, structure):
+    """
+    Parameters
+    ----------
+    structure: Raw text data input as a string
+
+    Returns
+    -------
+    obj.X_Sites: Node features
```

Comment:
The actual return type for this method is a dictionary. Could you adjust the return signature to match that?